csu

```diff
@@ -311,14 +308,6 @@ cmd_clean() {
   echo "If the following commands return an argument not found error,"
   echo "this is because there is nothing to delete for clean up."
 
-  echo
-  echo "Deleting unused volumes..."
```

Comment:
> Remove deletion of volumes as this deletes a Docker volume creating for storing node modules. Tidying Docker should be a user choice/action, not a project action.

This is the clean command, which is exactly the 'user choice/action' you want removing volumes to be isn't it?

But more to the point, I don't understand the justification for removing this both here and in the end command, considering:
- We added the removal of volumes because they filled the allocated disk space of my PC to the point where I couldn't log into linux anymore
- It's supposed to be "Deleting **unused** volumes..."
- I assume this PR is based on how it works in CSFG, which has no problem (volumes are removed in the `./csfg end` command, but not the `clean` one surprisingly)

so
- Don't remove this from the clean command
- Please explain why it should be removed from the end command

Reply:
Ah I see, I was following [Jack's PR](https://github.com/uccser/cs-field-guide/pull/731/files) without double checking :paddlin: I will add back in the code for deleting unused docker volumes in both the `end` and `clean` commands.